contrib/instrumentation_nightly.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 adb logcat -c
-adb logcat *:E -v color &
+adb logcat -- *:E -v color &
```

Comment:
To ensure consistency across all instances, please utilize the command `adb logcat ./*:E -v color &` as specified in the `instrumentation.sh` script.